3DM_RFC/RIW2021_RFC_Optimistic ZKC(S)P.md

```diff
@@ -0,0 +1,59 @@
+# RIW2021 | RFC: Optimistic ZKC(S)P
+
+_Status:_ **draft**; **~~ready for review~~**; **~~ready to publish~~ **
+
+_Area of Improvement:_ Data Delivery Metering
+
+_Estimated Effort Needed:_ &lt;?>
+
+_Prerequisite(s):_ &lt;?>
+
+_Priority:_ &lt;? P0, P1, P2>
+
+
+### Abstract
+
+In this RFC, we propose a way to rely on the ZKCP only for resolving a dispute when one of the parties decides to go rogue at the end, only incurring the cost of issuing the proofs in case of misbehavior.
+
+Note: Watch the full recording with Davidad’s presentation as it is a very complete description (LINK HERE)
```

Comment:
@davidad would you be ok in sharing presentation for this RFC?

Reply:
I just watched the [recording](https://drive.google.com/file/d/19D9Ww28UysW6RoCCpVKXxgFvX-dL9ZT9/view?usp=drivesdk) of this talk on OZKCP. I’m happy to say I’m actually proud of it and would be happy for it to be published on YouTube or wherever as appropriate. (Btw big thanks to @daviddias for graciously letting me run almost 50% longer than the assigned slot!)